pages/common/valgrind.md

```diff
@@ -0,0 +1,20 @@
+# valgrind
+
+> Wrapper for a set of expert tools for profiling, optimizing and debugging programs.
+> Commonly used tools include `memcheck`, `cachegrind`, `callgrind`, `massif`, `helgrind`, and `drd`.
+
+- Use the (default) Memcheck tool to show a diagnostic of memory usage by `program`:
```

Comment:
In the Valgrind manual, the tools are referred to by their capitalized proper names in the text while they are lowercase in the code.